azure-stack/hci/concepts/system-requirements-23h2.md

```diff
@@ -41,6 +41,9 @@ Here are the Azure requirements for your Azure Stack HCI cluster:
 
 Before you begin, make sure that the physical server and storage hardware used to deploy an Azure Stack HCI cluster meets the following requirements:
 
+> [!NOTE]
+> Single servers must use only a single drive type: Non-volatile Memory Express (NVMe) or Solid-State (SSD) drives.
+
```

Comment:
The placement of the above note should be in the table in the following row: 

|Data drives|At least 2 disks with a minimum capacity of 500 GB (SSD or HDD).<br>Single servers must use only a single drive type: Non-volatile Memory Express (NVMe) or Solid-State (SSD) drives.|